docs/book/developer-guide/steps-pipelines/inspecting-pipeline-runs.md

```diff
@@ -40,7 +40,7 @@ The repository contains a collection of all created pipelines with at least one
 run sorted by the time of their first run from oldest to newest.
 
 You can either access this collection via the `get_pipelines()` method or query
-a specific pipeline by name using `get_pipeline(pipeline_name=...)`:
+a specific pipeline by name using `get_pipeline(pipeline=...)`:
```

Comment:
there is nothing in the docs yet about the various new ways of accessing pipelines and steps. I found @AlexejPenner 's PR description in #733 extremely helpful. Could you perhaps replicate that information here somehow ?

Reply:
@AlexejPenner already changed it in the docs, but I agree, this PR description is actually really cool and much more comprehensive, I'll update the docs accordingly!